spring-boot/admin/src/main/java/eu/coatrack/admin/controllers/UserController.java

```diff
@@ -54,78 +53,82 @@
 @Controller
 @RequestMapping(value = "/")
 public class UserController {
-    
+
     private static final Logger log = LoggerFactory.getLogger(UserController.class);
-    
+
     @Autowired
     private UserRepository userRepository;
-    
+
     @Autowired
     private UserValidator userValidator;
-    
+
     @ModelAttribute("user")
     public User getUserObject() {
         return new User();
     }
-    
+
     @RequestMapping("/register")
     public String registerForm(User user, BindingResult bindingResult, Model model) {
-        
+
         if (user == null) {
             user = getUserObject();
         }
-        
+
         model.addAttribute("user", user);
-        
+
         return "register";
     }
-    
+
     @Value("${ygg.mail.sender.user}")
     private String mail_sender_user;
-    
+
     @Value("${ygg.mail.sender.password}")
     private String mail_sender_password;
-    
+
+    @Value("${ygg.mail.server.url:smtp.gmail.com}")
```

Comment:
I would suggest to remove the default values here (url and port), as then the application will not start-up in case of wrong configuration. With default values, it will start-up and wrong configuration would only lead to errors at runtime when a new user registers.

Reply:
done